intro.tex

```diff
@@ -1,55 +1,67 @@
-The total amount science validation time is currently planned for 8 weeks.
-As Rubin construction moves through the challenging phase of System Integration, Test and Commissioning (SIT-COM), on--sky time could be further reduced. 
-The Operations team is thus planning for various outcomes that might require special attention to producing \es opportunities in the first part of regular operations to ensure the community has prompt access to science-ready data products while the survey begins its relentless coverage of the sky leading to DR1.
+The total amount science validation time currently planned during commissioning is 8 weeks. 
+As Rubin construction moves through the challenging phase of System Integration, Test and Commissioning (SIT-COM), on--sky time could be further reduced.
+
+The Operations team is tracking the progress of the commissioning activities as they relate to \es opportunities to ensure that the community has prompt access to science-ready data products while the survey begins its relentless coverage of the sky leading to DR1.
+
+We broadly envisage two possible scenarios emerging from the commissioning phase of the construction project: 
 
-In all cases, it is assumed that the Rubin Construction project delivers an integrated system that can capture, transfer and process science-quality data at the time Operations begins.
-Planning will then consider the following three high-level options to ensure \es:
 \begin{itemize}
-\item \textbf{Option A}: 
-Science validation is completely successful.
-Begin the LSST working towards DR1 while providing \es data products. 
-\item \textbf{Option B:} 
-On-sky time in commissioning was reduced such that fewer science-ready data products were produced before the LSST begins but the Rubin system can capture and produce science-quality data. 
-Instigate a dedicated Early Science observing campaign for a limited period (3-6 months) at the start of full operations that is different to the regular survey cadence. 
-\item \textbf{Option C:} 
-The Rubin System  passes the construction completion requirements and can capture and produce science-quality data but the operations team is not yet ready to begin the LSST. 
-Further shakedown of operations procedures and data taking is required, which will result in a delay to the start of full survey operations. 
-This option will encompass elements of options B and C as appropriate. 
-\end{itemize}
+\item \textbf{Scenario A}:
+The full commissioning plan comprising system optimization and science validation is successfully executed as planned. 
+The Operations team then carries out an Operations Readiness Review (ORR) to effectively conduct a "full dress rehearsal" of science operations and demonstrate the readiness of the Operations team to execute the 10-year survey. 
```

Comment:
The ORR doesn't conduct a dress rehearsal.  Do you mean that we'd report on a dress rehearsal at ORR?

Reply:
No - that was not what I was thinking but I realise now that this not clear - will rephrase

Reply:
Replaced with "Operations Rehearsal and Readiness Review (ORR) to effectively conduct a "full dress rehearsal" of science operations and demonstrate the readiness of the Operations team to execute the 10-year survey. " 

- I mean to state that Ops will do an operations rehearsal .. then an (probably internal) review of our state of readiness